### ISSUE TITLE
Bots are out! Unpatched

### ISSUE THREAD

**Issue by 2368432172** [davidmann4/ogar-feeder-bot#579]
Hi i saw many bots coming out today including gbots and bagar.io. Im wondering when will the free bots be out. Thanks so much!

**Comment by idkRage** [comment]
i know how to get ip but i dont know how to update script so you can paste ip in
help!

**Comment by David1ali12** [comment]
agar.io give its protocol to ogar private server so this means its easy to fix we just need to use packets of ogar project I cant fix why I'm not a coder , bagario use news packets which ogar project have but as soon as ogar feeder unpatched bots they will patch again :(
https://github.com/OgarProject/Ogar/tree/master/src/packet

**Comment by agarioBotDev** [comment]
@David1ali12 as you may have noticed, agar.io changes it protocoll daily - with the new structure and the game client download it got easy for them. If you give me packets, I can do the packets stuff in minutes....

BUT ogar don't include the new packets.

**Comment by David1ali12** [comment]
@agarioBotDev  what about multi ogar it can work with different packets 
take a look https://github.com/Barbosik/MultiOgar/

**Comment by agarioBotDev** [comment]
@David1ali12 nope - agar.io packets are srambled, protocol is like version 5 in the core. Ogar works, because if ip is defined in url and don't exist in their databases agar.io accept the old, unscrambled packets. Bagar.io found out how to "translate" the packets.

**Comment by RaxBoon** [comment]
@agarioBotDev  You speak spanish

**Comment by agarioBotDev** [comment]
@RaxBoon Nope. I have everything ready on client side - map coords, player coords, server ip and party code. The scrambled packets make problems on bot side. Would be appriciated for any help from smart coders.

PS: i speak german

**Comment by Ben-Lazrek-Yassine** [comment]
Sooo RIP ogar bots ? there is no chance that they'll come back ?

**Comment by RaxBoon** [comment]
@agarioBotDev tu skype cual es

**Comment by mastkilxp** [comment]
what packet are we talking here about excactly?

**Comment by ImHITMan** [comment]
@bangana What do you mean "RIP Ogar bots"?, I have a clientside version of ogar bots which work for Ogar unlimited, To make it work for Mgar, All I'd really have to do is change some opcodes and done.
